File: weather_script.js
const apiKey = 'YOUR_API_KEY';
const city = 'London';
const apiUrl = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${apiKey}&units=metric`;

fetch(apiUrl)
  .then(response => response.json())
  .then(data => {
    console.log(data);
    const weatherDescription = data.weather[0].description;
    const temperature = data.main.temp;
    const humidity = data.main.humidity;
    const windSpeed = data.wind.speed;

    console.log(`Weather in ${city}:`);
    console.log(`Description: ${weatherDescription}`);
    console.log(`Temperature: ${temperature}°C`);
    console.log(`Humidity: ${humidity}%`);
    console.log(`Wind Speed: ${windSpeed} m/s`);
  })
  .catch(error => console.error('Error fetching the weather data:', error));

File: weather_script.py
import requests

api_key = 'YOUR_API_KEY'
city = 'London'
api_url = f'http://api.openweathermap.org/data/2.5/weather?q={city}&appid={api_key}&units=metric'

try:
    response = requests.get(api_url)
    response.raise_for_status()
    data = response.json()

    weather_description = data['weather'][0]['description']
    temperature = data['main']['temp']
    humidity = data['main']['humidity']
    wind_speed = data['wind']['speed']

    print(f'Weather in {city}:')
    print(f'Description: {weather_description}')
    print(f'Temperature: {temperature}°C')
    print(f'Humidity: {humidity}%')
    print(f'Wind Speed: {wind_speed} m/s')

except requests.exceptions.RequestException as error:
    print(f'Error fetching the weather data: {error}')